protos/validator.proto

```diff
@@ -33,59 +33,63 @@ message Message {
         TP_REGISTER_REQUEST = 1;
         // Registration response from the validator to the transaction processor
         TP_REGISTER_RESPONSE = 2;
+        // Tell the validator that the transaction processor won't take any more transactions
```

Comment:
This reordering of the enums is certainly fine here, but we should probably come up with a strategy for giving ourselves room to grow.